src/gui/DatabaseWidget.cpp

```diff
@@ -322,6 +322,11 @@ void DatabaseWidget::setPreviewSplitterSizes(const QList<int>& sizes)
     m_previewSplitter->setSizes(sizes);
 }
 
+void DatabaseWidget::setSearchStringForAutoType(const QString& search)
+{
+    m_searchStringForAutoType = search;
```

Comment:
This is a sign of an antipattern but there isn't much we can do about it without a total overhaul.